asset_sources/scss/_apidocs.scss

```diff
@@ -8,10 +9,14 @@
 
     & > li {
         list-style-type: none;
+        padding-top: 0 !important;
```

Comment:
I find `!important` to be quite confusing in practice, but I guess it's the simplest solution here?

Reply:
Yes unfortunately. The reason is that the selector has the same specificity as another selector, but the other one appears later in the CSS file. The clean solution would be to restructure the CSS rules accordingly, but for now ` !important` should be OK.